### PR TITLE
Minor improvements to Docker implementation

### PR DESCRIPTION
**Pull request by sirredbeard**
See comments.

### CODE REVIEW

File: linux_files/setup [hunk 1]
```diff
@@ -722,6 +722,9 @@ fi
 function dockerinstall_build_relay {
 
     #Build the relay
+
```
Comment (crramirez) [inline linux_files/setup]: The problem is wslupath has hardcoded the root in /mnt/c. If the user or wlinux-setup dockerinstall change the root, wslupath will return the wrong path.

File: linux_files/setup [hunk 2]
```diff
@@ -790,6 +793,10 @@ EOF
     sudo addgroup docker
     sudo adduser ${USER} docker
 
+    # can we delete the 'sources' folder left in wslupath -H?
```
Comment (crramirez) [inline linux_files/setup]: Yes I don't realize that a sources folder is left

File: linux_files/setup [hunk 3]
```diff
@@ -790,6 +793,10 @@ EOF
...
+
+    # can we hide the 'go' folder left in wslupath -H? (if necessary calling attrib +h)
```
Comment (crramirez) [inline linux_files/setup]: Yes we can

File: linux_files/setup [hunk 4]
```diff
@@ -5,7 +5,7 @@ CURDUR=""
 TMPDIR=""
 VERSION="1.1.25"
 GOVERSION="1.11.2"
-DOCKERVERSION="18.06.1-ce"
+DOCKERVERSION="18.09.0"
```
Comment (crramirez) [inline linux_files/setup]: I need to test everything before a version upgrade. Give me some time to check it

File: linux_files/setup (hunk 2 above)
Comment (crramirez) [reply linux_files/setup]: Eh, I cannot find the sources folder. The only one is left is in the tmp folder which is deleted at the end

**Comment by sirredbeard** [comment]
Submit a new PR and we will close this one.

**Pull request closed without merging**